### PR TITLE
measures/measures_detail_page: add comma between fields and restructure section inline w…

### PR DESCRIPTION
**Pull request by philli-m**
…ith a11y and adjust contact classes

### CODE REVIEW

File: apps/measures/templates/apps_measures/measures_detail_page.html [hunk 1]
```diff
@@ -48,18 +48,31 @@ <h1>{{ page.page_title }}</h1>
...
+                        {{ element }},
+                    {% endwith %}
+                {% endif %}
+
+                {% if page.facilitative_administration %}
+                    {% with element=page.get_facilitative_administration_display %}
+                        {{ element }},
+                    {% endwith %}
+                {% endif %}
+
```
[the remark this answers is not in the document]
Comment (philli-m) [reply apps/measures/templates/apps_measures/measures_detail_page.html:81]: @fuzzylogic2000 @Rineee this is alot worse right? couldn't get it to work with default but I am probably missing knowledge

[the remark this answers is not in the document]
Comment (Rineee) [reply apps/measures/templates/apps_measures/measures_detail_page.html:81]: What do you mean with default?
It seems very complicated, yes.. I could try and provide you that list from backend, would that help?

**Comment by Rineee** [comment]
@philli-m This would be ready, what do you think?

**Review by fuzzylogic2000 — APPROVED** [review]
Nice!
Should we also use it on the tiles? https://github.com/liqd/digitalstrategie/blob/a0d31b02593f11d6fab6739a891bc71c1123d0bd/apps/measures/templates/apps_measures/includes/measures_list_item.html#L20

**Comment by Rineee** [comment]
> Nice! Should we also use it on the tiles?
> 
> https://github.com/liqd/digitalstrategie/blob/a0d31b02593f11d6fab6739a891bc71c1123d0bd/apps/measures/templates/apps_measures/includes/measures_list_item.html#L20

It is not like that in the design: https://app.zeplin.io/project/61cddb8e8de87f61376d8eef/screen/642ec66ade276b6d042eeb47
Although it would probably make sense? Should I add it @fuzzylogic2000 ?

**Comment by fuzzylogic2000** [comment]
> > Nice! Should we also use it on the tiles?
> > https://github.com/liqd/digitalstrategie/blob/a0d31b02593f11d6fab6739a891bc71c1123d0bd/apps/measures/templates/apps_measures/includes/measures_list_item.html#L20
> 
> It is not like that in the design: https://app.zeplin.io/project/61cddb8e8de87f61376d8eef/screen/642ec66ade276b6d042eeb47 Although it would probably make sense? Should I add it @fuzzylogic2000 ?

But it is there in the design and on the tiles? In the design with commas and in the current version only with commas when they are from the same set. So yes, let's use this nice fix also for the tiles! @Rineee

**Comment by Rineee** [comment]
> > > Nice! Should we also use it on the tiles?
> > > https://github.com/liqd/digitalstrategie/blob/a0d31b02593f11d6fab6739a891bc71c1123d0bd/apps/measures/templates/apps_measures/includes/measures_list_item.html#L20
> > 
> > 
> > It is not like that in the design: https://app.zeplin.io/project/61cddb8e8de87f61376d8eef/screen/642ec66ade276b6d042eeb47 Although it would probably make sense? Should I add it @fuzzylogic2000 ?
> 
> But it is there in the design and on the tiles? In the design with commas and in the current version only with commas when they are from the same set. So yes, let's use this nice fix also for the tiles! @Rineee

Are we talking about the same thing? I dont see any commas separating the Handlungsfelder in the design :eyes: Each category is in its own line. But I will add it anyway then!

**Comment by fuzzylogic2000** [comment]
> > > > Nice! Should we also use it on the tiles?
> > > > https://github.com/liqd/digitalstrategie/blob/a0d31b02593f11d6fab6739a891bc71c1123d0bd/apps/measures/templates/apps_measures/includes/measures_list_item.html#L20
> > > 
> > > 
> > > It is not like that in the design: https://app.zeplin.io/project/61cddb8e8de87f61376d8eef/screen/642ec66ade276b6d042eeb47 Although it would probably make sense? Should I add it @fuzzylogic2000 ?
> > 
> > 
> > But it is there in the design and on the tiles? In the design with commas and in the current version only with commas when they are from the same set. So yes, let's use this nice fix also for the tiles! @Rineee
> 
> Are we talking about the same thing? I dont see any commas separating the Handlungsfelder in the design eyes Each category is in its own line. But I will add it anyway then!

No, you are right! They don't have commas. But then in the template it's currently wrong, because some do have commas. Let's just add them and ask @mcastro-lqd if we can do it like that or if we need to do it differently.

**Comment by Rineee** [comment]
> > > > > Nice! Should we also use it on the tiles?
> > > > > https://github.com/liqd/digitalstrategie/blob/a0d31b02593f11d6fab6739a891bc71c1123d0bd/apps/measures/templates/apps_measures/includes/measures_list_item.html#L20
> > > > 
> > > > 
> > > > It is not like that in the design: https://app.zeplin.io/project/61cddb8e8de87f61376d8eef/screen/642ec66ade276b6d042eeb47 Although it would probably make sense? Should I add it @fuzzylogic2000 ?
> > > 
> > > 
> > > But it is there in the design and on the tiles? In the design with commas and in the current version only with commas when they are from the same set. So yes, let's use this nice fix also for the tiles! @Rineee
> > 
> > 
> > Are we talking about the same thing? I dont see any commas separating the Handlungsfelder in the design eyes Each category is in its own line. But I will add it anyway then!
> 
> No, you are right! They don't have commas. But then in the template it's currently wrong, because some do have commas. Let's just add them and ask @mcastro-lqd if we can do it like that or if we need to do it differently.

I added them now on the tiles! And actually saw, that on the detail pages, its also without comma in the design.. so lets just see what he says when testing?

**Pull request merged**